thrift/tagEvent.thrift

```diff
@@ -0,0 +1,20 @@
+include "tag.thrift"
+
+namespace scala com.gu.tagmanagement
+
+enum EventType {
+    CREATE = 0,
```

Comment:
Recently we decided to not add a `CREATE` type for both atoms and content as in practice, updates and creation are handled the same way.